src/main/java/com/redhat/red/build/finder/BuildFinder.java

```diff
@@ -70,27 +69,47 @@
 
     private File outputDirectory;
 
+    private MultiValuedMap<String, Integer> checksumMap;
+
+    private List<String> archiveExtensions;
+
     public BuildFinder(ClientSession session, BuildConfig config) {
         this.session = session;
         this.config = config;
-        this.builds = new HashMap<>();
         this.outputDirectory = new File("");
+        this.checksumMap = new ArrayListValuedHashMap<>();
+
+        initBuilds();
     }
 
-    public Map<Integer, KojiBuild> findBuilds(Map<String, Collection<String>> checksumTable) throws KojiClientException {
-        if (checksumTable == null || checksumTable.isEmpty()) {
-            LOGGER.warn("Checksum table is empty");
-            return Collections.emptyMap();
-        }
+    private void initBuilds() {
+        builds = new HashMap<>();
+
+        KojiBuildInfo buildInfo = new KojiBuildInfo();
 
-        final Instant startTime = Instant.now();
+        buildInfo.setId(0);
+        buildInfo.setPackageId(0);
+        buildInfo.setBuildState(KojiBuildState.COMPLETE);
```

Comment:
Out of curiosity why is the state 0 marked as complete?

Reply:
There used to be code that would remove builds where `getBuildState() != KojiBuildState.COMPLETE`. I think that the default state would be `KojiBuildState.ALL(0)`. Leaving the state as 0 in the current code is probably safe. Anything interested in a real build should ignore build 0 (or `id <= 0`) anyway.